### ISSUE TITLE
Unstackable items form base game and mods get stuck in AE2 ME System

### ISSUE THREAD

**Issue by Boerny41** [Cazadorsniper/PO3#1730]
# Use the search function before reporting to see if the issue has been reported before.
**Pack Version:** 3.4.11F
**Pack Mode (Normal, Titan, Kappa, Mythic):** Kappa
**Singleplayer, Multiplayer, or LAN?:** Singleplayer

**Description:** Unstackable items get stuck in ME System. For example filled buckets (water/lava), vanilla iron pickaxe, woot power cell

**Steps to reproduce:** put multiple water buckets in ME System, open a recipe that uses said bucket, the number of buckets will go to one inside the ME System and that one item will be stuck.

**Crash log (use pastebin or issue will be closed):** /

**Comment by Chaise91** [comment]
Can also confirm this issue exists for anything that holds a charge in addition to being unstackable. Makes AE crafting basically unusable for devices with batteries.

**Comment by Cazadorsniper** [comment]
You can try updating AE2 to see if that fixes it.

**Comment by Chaise91** [comment]
> You can try updating AE2 to see if that fixes it.

Updated from 54.19 to 55.12 and unfortunately the problem persists. A water bucket is in my AE system but anytime I want to use it in a recipe, it is not recognized nor can it be pulled out. Interestingly, I am able to put it in/pull it out normally when I first logon but that doesn't last.

**Comment by Cazadorsniper** [comment]
Yeah that is kinda weird, maybe try adding fluid storage/crafting. Now the bug to pull that stuff out has been there for a long time and the only way to get them out is to use a fuzzy card.
